openedx_events_sender/receivers.py

```diff
@@ -0,0 +1,39 @@
+"""
+Handlers for `openedx-events` signals.
+
+Idea: when we add support for more signals, we should consider subscribing to all signals from `openedx-events`, and
+handling only the ones defined in settings. This way, we could change `send_enrollment_data` to a generic `send_data`.
+"""
+
+import logging
+from typing import Any, Dict, Optional
+
+from django.conf import settings
+from openedx_events.learning.data import CourseEnrollmentData
+
+from openedx_events_sender.tasks import send_data_task
+from openedx_events_sender.utils import prepare_data
+
+log = logging.getLogger(__name__)
+
+
+def send_enrollment_data(enrollment: CourseEnrollmentData, **_kwargs):
+    """
+    Send enrollment data after receiving the `COURSE_ENROLLMENT_CHANGED` signal.
+    """
+    if not (url := getattr(settings, "EVENT_SENDER_ENROLLMENT_URL", None)):
+        return
+    headers = getattr(settings, "EVENT_SENDER_ENROLLMENT_HEADERS", {})
+    field_mapping = getattr(settings, "EVENT_SENDER_ENROLLMENT_FIELD_MAPPING", None)
+
+    _send_data(enrollment, url, headers, field_mapping)
```

Comment:
I was thinking should we pass the event name here and include it as part of the payload, since in future this plugin could be used for multple events.
Although the url setting `EVENT_SENDER_ENROLLMENT_URL` is specific to the event, but the receiver system could have a unified url for multiple events (ex. third partly analytics, event logging, etc.), in which case having the event name in the payload could be useful.

Reply:
Good idea 👍🏻